docs/modules/mysql.md

```diff
@@ -1,9 +1,10 @@
 # MySQL
 
 ### Purpose
+[MySQL](https://www.mysql.com/) is a database management system. MySQL will organize and provide access to databases where sites can store information.
```

Comment:
Does it make sense to explicitly state that sites can store information in the databases? What if a supporting utility or module needs to use a database? It might be better to adjust the language in that section. Additionally, maybe you can elaborate on what type of DBMS MySQL is, or possibly investigate the other modules in this project and see if any of them are going to use MySQL that we can link back to.